spec/factories/districts.rb

```diff
@@ -3,6 +3,7 @@
     sequence :name do |n|
       "district#{n}"
     end
+    region { 'us-east-1' }
```

Comment:
Why is this needed?

Reply:
As the factory_bot's changelog explains, the default strategy has been changed so that associations are by default "built" instead of "create". Previously associations were created and [this method https://github.com/degica/barcelona/blob/master/app/models/district.rb#L177] is called before validation. But now the method is not called since associations are just built.

The other spec change is made for the same reason.

Reply:
I see, thanks!